lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -1321,7 +1296,8 @@ def deploy_vm(self):
         self.configure_cpu_and_memory(vm_obj=vm_obj, vm_creation=True)
         self.configure_disks(vm_obj=vm_obj)
         self.configure_network(vm_obj=vm_obj)
-        if self.should_deploy_from_template() and len(self.params['customizations']) > 0:
+
+        if self.should_deploy_from_template() and len(self.params['customization']) > 0:
```

Comment:
With this we can't customize from scratch vm. I think we only need:
if len(self.params['customization']) > 0: